### DISCUSSION TITLE
Translation to Dutch

### DISCUSSION THREAD

**Discussion by Neimod2004** [shaked6540/YoutubePlaylistDownloader#53] [Ideas]
Hello shaked6540,
First of all, I love your project.
If you are interested, could translate your app to Dutch if you would like that.
If you like it, contact me and send me a list of all the words and senteces that need to be translated.

Greetings,

Neimod

PS: If my English is bad that is because I natively  speek Dutch, but I understand English good enough.

**Comment by shaked6540** [comment]
Hi! If you want to translate the program to dutch, all you have to do is download [this file](https://github.com/shaked6540/YoutubePlaylistDownloader/blob/master/YoutubePlaylistDownloader/Languages/English.xaml) and then translate the text between the tags (only the text, not the x:Key part). 

If you want to see an example of a translated file, it should look like [this](https://github.com/shaked6540/YoutubePlaylistDownloader/blob/master/YoutubePlaylistDownloader/Languages/%D7%A2%D7%91%D7%A8%D7%99%D7%AA.xaml)

You can edit it with any text editor you would like

After you're done you can either start a pull request or send the file here and I will update it

Thanks!
